src/libs/thorvg/tvgCapi.cpp

```diff
@@ -141,6 +141,12 @@ TVG_API Tvg_Result tvg_canvas_sync(Tvg_Canvas* canvas)
     return (Tvg_Result) reinterpret_cast<Canvas*>(canvas)->sync();
 }
 
+TVG_API Tvg_Result tvg_canvas_set_viewport(Tvg_Canvas* canvas, int32_t x, int32_t y, int32_t w, int32_t h)
```

Comment:
Let's upstream this change to ThorVG.

Reply:
thorvg  already fixed yet

Reply:
https://github.com/thorvg/thorvg/commit/178cf2944d5d5176a80f32df3916aab866935dbc